### PR TITLE
Add page for Universal access

### PR DESCRIPTION
**Pull request by jancborchardt**
Documenting some basic and some more advanced means of universal access we support.

Please review @nextcloud/accessibility @peterdamian @tyrylu @skjnldsv @wobben 

I didn’t manage to get a full build setup running so I created the page "dry" – please let me know if I missed anything.

### CODE REVIEW

**Review by tyrylu — APPROVED** [review]
Looks good to me.

File: user_manual/universal_access.rst [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+* **High contrast theme:** A high contrast mode to ease your navigation. Visual quality will be reduced but clarity will be increased.
+* **Dark theme:** A dark theme to ease your eyes by reducing the overall luminosity and brightness. It is still under development, so please report any issues you may find.
+* **Dyslexia font:** OpenDyslexic is a free typeface/font designed to mitigate some of the common reading errors caused by dyslexia.
+
+To reach the accessibility settings:
+
+1. Open the settings menu at the end of the header
+2. Pick **Settings**
+2. In the navigation, pick **Accessibility**
```
Comment (MarcoZehe) [inline user_manual/universal_access.rst:78]: This should be 3.
```suggestion
3. In the navigation, pick **Accessibility**
```

**Review by MarcoZehe — APPROVED** [review]
Approved with my suggested change. This also produced a warning right there.

File: user_manual/universal_access.rst [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+We offer several themes you can activate to aid accessibility:
+
+* **High contrast theme:** A high contrast mode to ease your navigation. Visual quality will be reduced but clarity will be increased.
+* **Dark theme:** A dark theme to ease your eyes by reducing the overall luminosity and brightness. It is still under development, so please report any issues you may find.
+* **Dyslexia font:** OpenDyslexic is a free typeface/font designed to mitigate some of the common reading errors caused by dyslexia.
+
+To reach the accessibility settings:
+
+1. Open the settings menu at the end of the header
+2. Pick **Settings**
```
Comment (rakekniven) [inline user_manual/universal_access.rst:77]: The ordering is not ok. 1,2,2 -> 1,2,3, or?

**Review by rakekniven — CHANGES_REQUESTED** [review]
See comments at source line

File: user_manual/universal_access.rst [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+Navigating via keyboard
+-----------------------
+
+You can navigate the web interface with keyboard only just like you can with the mouse:
+
+* ``Tab`` and ``Shift Tab`` to move between elements
+* ``Enter`` or ``Space`` to activate or open the element (depending on the type of element)
+* ``Escape`` to be used to close modals, popover menus, and file viewers
+* ``left arrow`` and ``right arrow`` to navigate between photos in the viewer
```
Comment (Valdnet) [inline user_manual/universal_access.rst:33]:
```suggestion
* ``Left arrow`` and ``Right arrow`` to navigate between photos in the viewer
```

File: user_manual/universal_access.rst [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+* ``Ctrl f`` to focus the search field
```
Comment (Valdnet) [inline user_manual/universal_access.rst:34]:
```suggestion
* ``Ctrl + F`` to focus the search field
```

File: user_manual/universal_access.rst [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+* ``Ctrl s`` to save changes in editors like Nextcloud Text
```
Comment (Valdnet) [inline user_manual/universal_access.rst:35]:
```suggestion
* ``Ctrl + S`` to save changes in editors like Nextcloud Text
```

File: user_manual/universal_access.rst [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+
+For quicker navigation, we offer 2 "skip links" at the beginning of the document which allow you to:
+
+* Skip to main content
+* Skip to navigation of app
+
+Nextcloud Talk has shortcuts which are also documented inside the settings of the app itself:
+
+* ``c`` to focus the message input field
```
Comment (Valdnet) [inline user_manual/universal_access.rst:44]:
```suggestion
* ``C`` to focus the message input field
```

File: user_manual/universal_access.rst [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+* ``Escape`` to unfocus the message input field to be able to use shortcuts
+* ``f`` to fullscreen the chat or call
```
Comment (Valdnet) [inline user_manual/universal_access.rst:46]:
```suggestion
* ``F`` to fullscreen the chat or call
```

File: user_manual/universal_access.rst [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+* While in a call:
+   * ``m`` to toggle the microphone on and off
```
Comment (Valdnet) [inline user_manual/universal_access.rst:48]:
```suggestion
   * ``M`` to toggle the microphone on and off
```

File: user_manual/universal_access.rst [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+   * ``v`` to toggle video on and off
```
Comment (Valdnet) [inline user_manual/universal_access.rst:49]:
```suggestion
   * ``V`` to toggle video on and off
```

File: user_manual/universal_access.rst [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+   * ``Space`` for push to talk or push to mute
+   * ``r`` to raise or lower hand
```
Comment (Valdnet) [inline user_manual/universal_access.rst:51]:
```suggestion
   * ``R`` to raise or lower hand
```

File: user_manual/universal_access.rst [hunk 11]
```diff
@@ -0,0 +1,83 @@
...
+
+Nextcloud Mail has shortcuts as well, also documented inside the settings of the app itself:
+
+* ``c`` to compose a new message
```
Comment (Valdnet) [inline user_manual/universal_access.rst:55]:
```suggestion
* ``C`` to compose a new message
```

File: user_manual/universal_access.rst [hunk 12]
```diff
@@ -0,0 +1,83 @@
...
+* ``Left arrow`` to switch to a newer message
+* ``Right arrow`` to switch to an older message
+* ``s`` to toggle a message as favorite
```
Comment (Valdnet) [inline user_manual/universal_access.rst:58]:
```suggestion
* ``S`` to toggle a message as favorite
```

File: user_manual/universal_access.rst [hunk 13]
```diff
@@ -0,0 +1,83 @@
...
+* ``u`` to toggle a message unread
```
Comment (Valdnet) [inline user_manual/universal_access.rst:59]:
```suggestion
* ``U`` to toggle a message unread
```

File: user_manual/universal_access.rst [hunk 14]
```diff
@@ -0,0 +1,83 @@
...
+* ``Del`` to delete a message
+* ``Ctrl Enter`` to send
```
Comment (Valdnet) [inline user_manual/universal_access.rst:61]:
```suggestion
* ``Ctrl + Enter`` to send
```

File: user_manual/universal_access.rst [hunk 15]
```diff
@@ -0,0 +1,83 @@
...
+* ``r`` to refresh and load new mails
```
Comment (Valdnet) [inline user_manual/universal_access.rst:62]:
```suggestion
* ``R`` to refresh and load new mails
```

File: user_manual/universal_access.rst [hunk 16]
```diff
@@ -0,0 +1,83 @@
...
+You can zoom in and out to fit the text and element size to your liking.
+The left navigation and right sidebar can be expanded or collapsed.
+
+
+Navigating via keyboard
+-----------------------
+
+You can navigate the web interface with keyboard only just like you can with the mouse:
+
+* ``Tab`` and ``Shift Tab`` to move between elements
```
Comment (Valdnet) [inline user_manual/universal_access.rst:30]:
```suggestion
* ``Tab`` and ``Shift + Tab`` to move between elements
```

File: user_manual/universal_access.rst [hunk 17]
```diff
@@ -0,0 +1,83 @@
...
+   * ``r`` to raise or lower hand
+
+Nextcloud Mail has shortcuts as well, also documented inside the settings of the app itself:
+
+* ``c`` to compose a new message
+* ``Left arrow`` to switch to a newer message
+* ``Right arrow`` to switch to an older message
+* ``s`` to toggle a message as favorite
+* ``u`` to toggle a message unread
+* ``Del`` to delete a message
```
Comment (Valdnet) [inline user_manual/universal_access.rst:60]:
```suggestion
* ``Del`` to delete a message
* ``Ctrl + F`` to focus the search field
```

**Comment by Valdnet** [comment]
I made suggestions for changes 😉.

File: user_manual/universal_access.rst (hunk 13 above)
Comment (rakekniven) [reply user_manual/universal_access.rst:59]: @Valdnet Why uppercase? See all other documents pleas.

Comment (Valdnet) [reply user_manual/universal_access.rst:59]: @rakekniven In my opinion, keyboard shortcuts should be capitalized. Just like it is in applications.

> See all other documents please.

If it is lowercase in other documents, this should also be changed 😉.

Comment (rakekniven) [reply user_manual/universal_access.rst:59]: Congratulations. You just found your next project 😄 .

Comment (Valdnet) [reply user_manual/universal_access.rst:59]: I guess it looks like it  :stuck_out_tongue_closed_eyes:.

Comment (jancborchardt) [reply user_manual/universal_access.rst:59]: Right – I was looking at https://docs.github.com/en/github/getting-started-with-github/keyboard-shortcuts where they lowercase everything but it indeed looks weird compared to https://support.apple.com/en-us/HT201236

Will change it as per your suggestions, as I also double checked the app settings and they use upper case as well.

Comment (jancborchardt) [reply user_manual/universal_access.rst]: And fixed :)

**Comment by jancborchardt** [comment]
Thanks for the review @MarcoZehe @rakekniven @Valdnet, I fixed all the issues now.

And since I don’t have a local documentation setup: Is it enough to add the page in `user_manual/contents.rst` like I did, or is anything else needed to have it show up?

**Review by rakekniven — APPROVED** [review]

**Review by Valdnet — APPROVED** [review]

File: user_manual/universal_access.rst [hunk 18]
```diff
@@ -0,0 +1,83 @@
...
+<https://github.com/nextcloud/server/issues/>`_. And if you want to get
+involved, come `join our design team <https://nextcloud.com/design>`_!
+
+
+Zoom and responsiveness
+-----------------------
+
+The Nextcloud interface is fully responsive and usable on any size of screen.
+You can zoom in and out to fit the text and element size to your liking.
+The left navigation and right sidebar can be expanded or collapsed.
```
Comment (nickvergessen) [inline user_manual/universal_access.rst:22]: Left and right are not to inclusive with RTL languages when they swap it at some point, is it needed?

Comment (jancborchardt) [reply user_manual/universal_access.rst:22]: Good catch, fix is at https://github.com/nextcloud/documentation/pull/6433

**Pull request merged**